checkmate/views/status.py

```diff
@@ -1,10 +1,10 @@
-"""The status end-point required to allow us to work with AWS."""
-
 from pyramid.view import view_config
+from sentry_sdk import capture_message
 
 
-@view_config(route_name="get_status", renderer="json", http_cache=0)
-def get_status(_request):
-    """Status endpoint."""
```

Comment:
I also removed this docstring because it's really pointless: it says nothing that the function name doesn't already say.